subjects/devops/calculator/README.md

```diff
@@ -0,0 +1,62 @@
+## Calculator
+
+### Instructions
+
+In this exercise you will make a script `calculator.sh` that will take 3 arguments, calculate the result and print it on the standard output.
+
+- The first argument and the third argument will be numbers.
+- The second argument will be the operator.
+
+Each operator should have its own function named as follow:
+- `+`: `do_add()`.
+- `-`: `do_sub()`.
+- `*`: `do_mult()`.
+- `/`: `do_divide()`.
+
+Each function will receive two arguments, the left number and the right number, and return the result of the operation.
+
+The functions assume the input is valid, so the input should be checked before to call the functions.
+
+> The functions will also be tested individually, so it is important to name each function exactly as above, the behavior of the functions have to match the exercise instructions.
+
+### Usage
+
+```console
+$ ./calculator.sh 20 "*" 3
+60
+$ ./calculator.sh 20 / 20
+1
+$ ./calculator.sh -1 - 10
+-11
+$
+```
+
+### Error handling
+
+All errors will print a specific message on **stderr** (ending with a newline) and returns a specific non-zero value:
+- Wrong number of arguments: `"Error: expect 3 arguments"`, returns `1`.
+- Division by 0: `"Error: division by 0"`, exit with `2`.
+- Invalid operator: `"Error: invalid operator"`, exit with `3`.
+- Invalid number(s): `"Error: invalid number"`, exit with `4`.
+
+> Negative numbers are also a valid input.
+
+### Hints
```

Comment:
Here in the hints, normally we are giving more info in the commands they will probably use, like an intro to what is the command an an example of usage.
In this piscine we are assuming that the students will not have programming background so we need to help as much as possible without giving the answer of course.
So in this case you can add an example of some commands usages that they need to know.

Reply:
I've added hints for the `getent` command, which is normally the only new command they will need to use.

Reply:
Yes we should only add info if it is something new like the `getent`, but maybe you forgot to push this change because i don't see any info on `getent` in the readme.

Reply:
i guess the `getent` is for the other exercise